src/three-components/Model.js

```diff
@@ -50,7 +50,10 @@ export default class Model extends Object3D {
 
   applyEnvironmentMap(map) {
     this.modelContainer.traverse(obj => {
-      if (obj && obj.isMesh && obj.material) {
+      // Unlit models (using MeshBasicMaterial) should not apply
+      // an environment map, even though `map` is the currently configured
+      // environment map.
+      if (obj && obj.isMesh && obj.material && !obj.material.isMeshBasicMaterial) {
```

Comment:
For some edge cases, `obj.material` could be an array of materials. We should probably avoid creating those in GLTFLoader at some point, but for now it's worth handling.